x-pack/legacy/plugins/siem/public/pages/detection_engine/create_rule/components/mitre/index.tsx

```diff
@@ -0,0 +1,167 @@
+
+  const removeItem = useCallback(
+    (index: number) => {
+      const values = field.value as string[];
+      field.setValue([...values.slice(0, index), ...values.slice(index + 1)]);
+    },
+    [field]
+  );
+
+  const addItem = useCallback(() => {
+    const values = field.value as IMitreEnterpriseAttack[];
+    const { id, name, reference } = tacticsOptions[0];
+    if (!isEmpty(values[values.length - 1])) {
+      field.setValue([...values, { tactic: { id, name, reference }, techniques: [] }]);
+    } else {
+      field.setValue([{ tactic: { id, name, reference }, techniques: [] }]);
+    }
+  }, [field]);
+
+  const updateTactic = useCallback(
+    (index: number, event: ChangeEvent<HTMLSelectElement>) => {
+      const values = field.value as IMitreEnterpriseAttack[];
+      const { id, reference, name } = tacticsOptions.find(t => t.value === event.target.value) || {
+        id: '',
+        name: '',
+        reference: '',
+      };
+      field.setValue([
+        ...values.slice(0, index),
+        {
+          ...values[index],
+          tactic: { id, reference, name },
+        },
+        ...values.slice(index + 1),
+      ]);
+    },
+    [field]
+  );
+
+  const updateTechniques = useCallback(
+    (index: number, selectedOptions: unknown[]) => {
+      field.setValue([
+        ...values.slice(0, index),
+        {
+          ...values[index],
+          techniques: selectedOptions,
+        },
+        ...values.slice(index + 1),
+      ]);
+    },
+    [field]
+  );
+
+  const values = field.value as IMitreEnterpriseAttack[];
+  const selectTacticOptions = useMemo(
+    () => tacticsOptions.map(t => ({ text: t.text, value: t.value })),
+    [tacticsOptions]
+  );
+  return (
+    <EuiFormRow
+      label={field.label}
+      labelAppend={field.labelAppend}
+      error={errorMessage}
+      isInvalid={isInvalid}
+      fullWidth
+      data-test-subj={dataTestSubj}
+      describedByIds={idAria ? [idAria] : undefined}
+    >
+      <>
+        {values.map((item, index) => {
+          const euiSelectFieldProps = {
+            disabled: isDisabled,
+          };
+          return (
+            <div key={index}>
+              <EuiFlexGroup gutterSize="xs" justifyContent="spaceBetween" alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiSelect
+                    id="selectDocExample"
+                    options={selectTacticOptions}
+                    aria-label=""
+                    onChange={updateTactic.bind(null, index)}
+                    prepend={I18n.TACTIC}
+                    compressed
+                    fullWidth={false}
+                    value={camelCase(item.tactic.name)}
+                    {...euiSelectFieldProps}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem grow={true}>
+                  <MyEuiFormControlLayout compressed fullWidth prepend={I18n.TECHNIQUES}>
+                    <EuiComboBox
```

Comment:
Bottom of this combobox is cut off! 😮 

![image](https://user-images.githubusercontent.com/2946766/70352936-827a4b80-1829-11ea-9f23-833daec9afa4.png)
 vs EuiComboBox from the [EUI Docs](https://elastic.github.io/eui/#/forms/combo-box)

![image](https://user-images.githubusercontent.com/2946766/70353022-af2e6300-1829-11ea-93ad-8b8036e975f5.png)

Reply:
This is a by product of prepending 'Technique' with `MyEuiFormControlLayout` 👍